lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9639,6 +9639,86 @@ namespace Js
         return superCtor;
     }
 
+    Var JavascriptOperators::OP_ImportCall(Var specifier, ScriptContext* scriptContext)
+    {
+        ModuleRecordBase *moduleRecordBase = nullptr;
+        SourceTextModuleRecord *moduleRecord = nullptr;
+
+        JavascriptFunction* pfuncCaller;
+        if (JavascriptStackWalker::GetCaller(&pfuncCaller, scriptContext) && pfuncCaller && pfuncCaller->IsScriptFunction())
```

Comment:
Why do we need to use the stack walker here?    Didn't the interpreter just called this OP_ImportCall and know what the function is already?  Can't the interpreter pass it in?